### ISSUE TITLE
Completions not working

### ISSUE THREAD

**Issue by markogresak** [TypeStrong/atom-typescript#270]
Hi!

I can't get completions to work. I have been using autocomplete-plus since I've started using atom (over 1 year ago) and it always worked fine. Now when I've installed atom-typescript, it stopped working for all typescript files.
I have tried using [atom-typescript-examples/autocomplete](https://github.com/TypeStrong/atom-typescript-examples/tree/master/autocomplete) as project, but it still didn't trigger autocomplete, not even when implicitly pressing <kbd>ctrl</kbd>+<kbd>space</kbd> (<kbd>cmd</kbd>+<kbd>space</kbd> triggers Spotlight).
If I open Command Palette an select Autocomplete: Toggle, the completion works as expected, but when selecting Typescript: Autocomplete, nothing happens. 

I have no idea what's wrong and where to start looking.
Can it be some package that is interfering with atom-typescript?

Here is list of my installed packages:
```
Zen@0.12.0
angularjs@0.1.0
atom-beautify@0.24.0
atom-bootstrap3@1.2.0
atom-terminal@0.7.0
atom-typescript@2.15.7
autocomplete-bibtex@0.5.6
autocomplete-html@0.5.0
autocomplete-paths@1.0.2
autocomplete-plus@2.10.0
autocomplete-plus-python-jedi@0.2.6
autocomplete-snippets@1.2.0
coffee-lint@0.7.3
coffee-refactor@0.6.2
color-picker@1.6.0
dash@1.1.1
docblockr@0.6.3
editor-stats@0.17.0
editorconfig@0.3.3
emmet@2.3.7
file-icons@1.5.4
git-plus@4.4.13
highlight-line@0.10.1
highlight-selected@0.9.1
isotope-ui@2.1.1
javascript-snippets@1.0.0
language-arm@0.3.3
language-csharp@0.5.0
language-cshtml@0.1.1
language-dart@0.1.1
language-latex@0.6.1
language-matlab-octave@0.1.1
language-pascal@0.5.0
language-restructuredtext@0.4.0
latex@0.21.0
linter@0.12.0
linter-coffeelint@0.2.1
linter-csslint@0.0.11
linter-jscs@1.10.0
linter-jshint@0.1.2
linter-less@0.3.1
linter-tidy@1.0.0
linter-tslint@0.2.1
markdown-pdf@1.3.5
merge-conflicts@1.3.1
monokai@0.14.0
monokai-soda@0.5.0
open-in-browser@0.4.3
open-last-project@0.4.1
open-terminal-here@0.5.0
pdf-view@0.19.0
preview@0.13.7
project-manager@1.15.5
refactor@0.4.1
script@2.19.0
seti-monokai@0.3.1
seti-ui@0.6.3
vim-mode@0.44.0
wakatime@4.0.2
you-complete-me@0.1.1
```

**Comment by markogresak** [comment]
After disabling and re-enabling each plugin, I've finally found out which is causing problems.

Unluckily, it was the [you-complete-me](https://atom.io/packages/you-complete-me), the last one on the list. I have removed it and the completions started working right away. I'm not sure if it's even worth looking for a fix, the plugin has only 300 users and I didn't find to be useful for me, I tried it because I liked the vim version.

**Comment by basarat** [comment]
Cool. I didn't start any investigation so saved me some time :rose:

**Comment by maxime1992** [comment]
I had the "turbo-javacript" module. I just disabled it and now typescript autocomplete is working well, thanks !
